src/Hangfire.Core/Email/EmailStorage.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hangfire.Email
+{
+    public class EmailStorage
+    {
```

Comment:
this should be in the email module as there should be no reference to emails at all in here